### DISCUSSION TITLE
Project Details in Summary View

### DISCUSSION THREAD

**Discussion by HunterDellere** [mcndt/obsidian-toggl-integration#56] [Feature Requests]
I'm using Toggl to bucket general time categories like productive and unproductive time and plan to use the Summary dataview to review against my targets. One detail I wish was more visible is the breakdown of time spent in a given day on the summary view. It would be nice to have the columns of Summary views configurable to show only cumulative time or time grouped by project and stacked.

**Comment by mcndt** [comment]
Hi Hunter. Interesting use case! It never ceases to amaze me the creative ways people use time tracking that I have never even thought about.

I'm quite intrigued because this sounds like something I'd like to experiment with in my time tracking as well.

Can you elaborate on what this would look like? Perhaps you can draw a quick mockup with MS Paint or [Excalidraw](https://excalidraw.com/).
